Earthfile

```diff
@@ -114,10 +120,12 @@ lint-scripts-auth-test:
     # flag to source setup.sh during analysis.
     RUN shellcheck -x test-*.sh
 
+# lint-scripts runs the spellcheck package against both misc and test shell scripts
 lint-scripts:
     BUILD +lint-scripts-auth-test
     BUILD +lint-scripts-misc
 
+# earthly-script-no-stdout validates the earthly script doesn't print anything to stdout (stderr only)
```

Comment:
this duplicates the comment below (and was likely written before `earthly docs` existed); can you remove the comment contained in the code block.

Additionally, I think it's valuable to reference it as `./earthly` because the term `earthly` can be rather generic.

```suggestion
# earthly-script-no-stdout validates the ./earthly script doesn't print anything to stdout (stderr only)
```